### PR TITLE
Slim codebase

### PR DESCRIPTION
**Pull request by MylesBartlett**

### CODE REVIEW

File: optimisation/loss.py [hunk 1]
```diff
@@ -20,34 +20,36 @@ class VGGLoss(nn.Module):
...
+    vgg_feature_layer_default = '22'  # VGG19 layer number from which to extract features
 
     def __init__(self, args=None, prefactor=0.006):
-        """
-        Args:
-            prefactor: prefactor by which to scale the loss.
-            Rescaling by a factor of 1 / 12.75 gives VGG losses of a scale that
-            is comparable to MSE loss. This is equivalent to multiplying with a
-            rescaling factor of ≈ 0.006.
-        """
```
Comment (tmke8) [inline optimisation/loss.py]: why delete the documentation?

File: main_gan.py [hunk 2]
```diff
@@ -107,8 +107,8 @@ def parse_arguments(raw_args=None):
                         help='use the class information of images')
 
     # VGG loss
-    parser.add_argument('--vgg_feature_layer', type=int, default=11,
-                        help='VGG19 layer number from which to extract features')
```
Comment (tmke8) [inline main_gan.py:111]: was it necessary to turn the type into string?

**Review by tmke8 — APPROVED** [review]
Only small things
